capstone.css

```diff
@@ -445,89 +445,89 @@ footer {
 
 @media only screen and (max-width: 768px) {
   .Hamburger {
-  display: inline-block;
-  height: 20px;
-  width: 20px;
+    display: inline-block;
+    height: 20px;
+    width: 20px;
   }
 
   .firsthead {
-  padding-bottom: 0;
+    padding-bottom: 0;
   }
 
   .firsthead h1 {
-  font-size: 30px;
+    font-size: 30px;
   }
 
   .program {
-  height: 930px;
+    height: 930px;
   }
 
   .logo {
-  display: none;
+    display: none;
   }
 
   .SocialBar {
- display: none;
+    display: none;
   }
 
   .navSmall {
- display: flex;
+    display: flex;
   }
 
   .All-Programs {
- grid-auto-flow: row;
- height: 900px;
- overflow-y: hidden;
+    grid-auto-flow: row;
+    height: 900px;
+    overflow-y: hidden;
   }
 
   .Aboutprogram p {
- height: 600px;
- overflow-y: hidden;
+    height: 600px;
+    overflow-y: hidden;
   }
 
   .Speaker {
- grid-template-columns: 1fr;
+    grid-template-columns: 1fr;
   }
 
   .NavBar {
- display: flex;
- flex-direction: column;
- height: 100%;
- width: 0;
- background-color: #272a31;
- color: white;
- position: fixed;
- top: 0;
- left: 0;
- transition: width 0.5s ease;
- overflow-x: hidden;
- z-index: 5;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    width: 0;
+    background-color: #272a31;
+    color: white;
+    position: fixed;
+    top: 0;
+    left: 0;
+    transition: width 0.5s ease;
+    overflow-x: hidden;
+    z-index: 5;
   }
 
   .Navlink {
- display: flex;
- flex-direction: column;
- justify-content: center;
- margin-top: 74px;
- width: 100%;
- font-weight: bolder;
- font-size: larger;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-top: 74px;
+    width: 100%;
+    font-weight: bolder;
+    font-size: larger;
   }
 
   .Navlink li {
- justify-content: left;
- border-bottom: 1px solid orangered;
- padding-bottom: 12px;
- width: 80%;
- margin: 10px auto;
+    justify-content: left;
+    border-bottom: 1px solid orangered;
+    padding-bottom: 12px;
+    width: 80%;
+    margin: 10px auto;
   }
 
   .Navlink li:last-child {
- color: orangered;
+    color: orangered;
   }
 
   .toggleMenu {
- width: 100%;
- overflow-x: hidden;
+    width: 100%;
+    overflow-x: hidden;
   }
```

Comment:
- [ ] Please implement the footer in the home page. I have attached some screenshots to show you what I mean 👇 👇 
**Given design**
![image](https://user-images.githubusercontent.com/90222110/191798827-5c700680-7f69-40bc-90af-97634185053d.png)

Reply:
Ok . I will do that.

Reply:
I added this in the footer section of about.html.